pkg/registries/registries.go

```diff
@@ -104,18 +110,31 @@ func mergedMirrorSets(icspRules []*apioperatorsv1alpha1.ImageContentSourcePolicy
 func EditRegistriesConfig(config *sysregistriesv2.V2RegistriesConf, insecureRegistries, blockedRegistries []string, icspRules []*apioperatorsv1alpha1.ImageContentSourcePolicy) error {
```

Comment:
The documentation about “whole registries” above should document that wildcards are accepted.

Reply:
Fixed